sigs.yml

```diff
@@ -364,7 +364,7 @@
     - name: 'Juraci Paixão Kröhling'
       github: jpkrohling
   - name: 'C++: SDK'
-    meeting: Every week alternating between Monday at 13:00 PT and Wednesday at 9:00
+    meeting: Alternating between Monday at 13:00 PT and Wednesday at 09:00 PT
```

Comment:
@open-telemetry/cpp-maintainers please confirm that the other meeting was also PT as the timezone was missing

Reply:
Confirmed, change looks correct.